src/components/tree/TreeMenu.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from "react";
+import { Props } from "../../types/treeMenuTypes";
+
+import "./TreeMenu.css";
+import Button from "./Button";
+
+
+const TreeMenu: FC<Props> = ({info, treeSections, show, toggleShow}) => {
```

Comment:
Seems like we don't need that `info` parameter.

Reply:
:+1: